include/mesh/parallel_mesh.h

```diff
@@ -42,10 +42,14 @@ class ParallelMesh : public DistributedMesh
   ParallelMesh (const UnstructuredMesh & other_mesh) : DistributedMesh(other_mesh) {}
 
   /**
-   * Use the equality operator of the base DistributedMesh class
-   * if an rvalue argument is provided.
+   * Provides move functionality via base class move assignment operator.
    */
-  using DistributedMesh::operator=;
+  virtual MeshBase & assign(MeshBase * other_mesh) override
+  {
+    DistributedMesh::assign(other_mesh);
+
+    return *this;
+  }
```

Comment:
Is this necessary?  Shouldn't we already be inheriting DistributedMesh::assign here?

Reply:
True, same for BoundaryMesh as well.